iceberg/iceberg-handler/src/test/results/positive/llap/iceberg_bucket_map_join_4.q.out

```diff
@@ -242,23 +242,23 @@ Stage-0
           PARTITION_ONLY_SHUFFLE [RS_38]
             Group By Operator [GBY_37] (rows=1 width=8)
               Output:["_col0"],aggregations:["count()"]
-              Map Join Operator [MAPJOIN_36] (rows=14380 width=8)
+              Map Join Operator [MAPJOIN_36] (rows=8751 width=89)
                 Conds:SEL_35._col0=RS_33._col0(Inner)
               <-Map 3 [BROADCAST_EDGE] vectorized, llap
                 BROADCAST [RS_33]
                   PartitionCols:_col0
-                  Select Operator [SEL_32] (rows=1476 width=4)
+                  Select Operator [SEL_32] (rows=3918 width=89)
                     Output:["_col0"]
-                    Filter Operator [FIL_31] (rows=1476 width=89)
+                    Filter Operator [FIL_31] (rows=3918 width=89)
                       predicate:(part is not null and key is not null)
-                      TableScan [TS_3] (rows=1476 width=89)
-                        default@srcbucket_mapjoin_part_2_n6,b,Tbl:COMPLETE,Col:COMPLETE,Output:["key","part"]
-              <-Select Operator [SEL_35] (rows=2952 width=4)
+                      TableScan [TS_3] (rows=3918 width=89)
+                        default@srcbucket_mapjoin_part_2_n6,b,Tbl:PARTIAL,Col:COMPLETE,Output:["key","part"]
```

Comment:
hi @okumin, after adding support in iceberg for basic partition stats, I've noticed that a few TableScans have PARTIAL stats. 
The reason is that the partition name seems to include the bucketing number
`srcbucket_mapjoin_part_1_n2(part=2/key_bucket_4=0)`, however, stats record doesn't have it `part=2/key_bucket=0`
Was it necessary to have the bucketing number in the name or it can be removed?